react-intro/react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,42 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+  It is a a development server that uses Webpack to compile React, JSX, and ES6, auto-prefix CSS files.
+- What is a React component?
+  A component is a piece of the UI (user interface) that has its own logic and appearance. A component can be as small as a button, or as large as an entire page.
+- How do you mount a React app (root component) to the DOM?
+  By creating a component and importing it.
```

Comment:
Use the react-dom package to create a root attached to an HTML element (usually a div)
Call root.render() and pass it the app's main component.